libqtile/widget/systray.py

```diff
@@ -170,6 +178,21 @@ def _configure(self, qtile, bar):
         )
         qtile.core._root.send_event(event, mask=EventMask.StructureNotify)
 
+        Systray._instances += 1
+
+    def create_mirror(self):
+        """
+        Systray cannot be mirrored as we do not use a Drawer object to render icons.
+
+        Return a fake widget which will crash during `_configure` so user will get a
+        ConfigErrorWidget in their bar instead.
+        """
+        class BrokenMirror(base._Widget):
+            def _configure(self, qtile, bar):
+                raise ConfigError("Systray widget cannot be mirrored.")
+
+        return BrokenMirror(0)
```

Comment:
7 years' bad luck! :wink:

Reply:
Is this necessary? I'm wondering, if you did just:
`return self`
Then it would run `_configure` on itself again and `ConfigError` that way. Might be slightly cleaner? Also consider when this happens, somebody might be like "OK, guess I'll make a second instance instead of passing the same one to two bars", then they hit a new config error. Thoughts?

Reply:
Oh, yeah. That's much neater!